BASE/Test/Microsoft.ApplicationInsights.Test/Shared/Metrics/TestUtility/StubApplicationInsightsTelemetryChannel.cs

```diff
@@ -1,6 +1,6 @@
 ﻿using System;
 using System.Collections.Generic;
-
+using System.Threading.Tasks;
```

Comment:
Looks like the coding style is trying to put a space between `System` and vendor namespaces.